app/react/Settings/components/specs/ToggleChildren.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { ToggleChildren } from '../ToggleChildren';
+
+describe('Toggle children', () => {
+  let component: ShallowWrapper<typeof ToggleChildren> = shallow(
+    <ToggleChildren toggled={false} />
+  );
+
+  it('should not display children if not toggled', () => {
+    expect(component.find('.toggle-children-children').text()).toBe('');
+  });
+
+  it('should display nothing if children are not passed', () => {
+    component = shallow(<ToggleChildren toggled />);
+    expect(component.find('.toggle-children-children').text()).toBe('');
+  });
+
+  it('should display children if toggled', () => {
+    component = shallow(
+      <ToggleChildren toggled>
+        <p>Content</p>
+      </ToggleChildren>
+    );
+    expect(component.find('.toggle-children-children').text()).toBe('Content');
+  });
```

Comment:
same as the previous test, the only reason why this shows content is because there is content, if you set toggled=false the test will still be green. this is because togglechildren does render the child and "hides" them via display:none

Reply:
Done